charts/opentelemetry-kube-stack/examples/prometheus-otel/values.yaml

```diff
@@ -35,7 +35,8 @@ collectors:
             name: otel-collector-secret
     presets:
       logsCollection:
-        enabled: false
+        enabled: true
+        includeCollectorLogs: false
```

Comment:
lets change an example that's already using logs collection, this example serves as a replacement for the kube-prom-stack chart.

Reply:
should we also test setting `collectorContainerNames`?